docs/modules/ROOT/pages/changelog.adoc

```diff
@@ -1,5 +1,21 @@
 = Changelog
 
+[[release-2024-09-05]]
+== September 5th, 2024
+
+* *General*
+** Feature: UI redesign and improvements for the Defender 2.0 dashboard.
+** Feature: Add support for the proposals pagination.
+** Feature: Add support for EIP 1559 for fuse network.
+** Feature: Improve resiliency of the relayers by retrying transactions with specific error responses from providers.
+** Feature: Allow EOAs to be used for upgrade approval process.
```

Comment:
specific to deploy?

Reply:
I'm combining everything under general, which we have been doing from past few releases...we can change it later if we see the need